plenum/common/batched.py

```diff
@@ -68,21 +69,16 @@ def send(self,
         # Signing (if required) and serializing before enqueueing otherwise
         # each call to `_enqueue` will have to sign it and `transmit` will try
         # to serialize it which is waste of resources
-        message_parts, err_msg = \
+        message_parts = \
```

Comment:
Do we have tests to check that we raise exceptions if couldn't split big message?